FAIL
Run started 'close-stdout' but it never finished
